src/event_store_ms/src/main/java/config/Config.java

```diff
@@ -0,0 +1,36 @@
+package config;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.ConnectionFactory;
+import java.sql.Connection;
+import java.io.IOException;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Config {
+
+    public static void setConfigFactory(ConnectionFactory factory) {
+        factory.setHost("localhost");
```

Comment:
adres musi być przekazywany przez zmienną środowiskową, to nie będzie działać na klastrze, bo tam adres rabbita to `rabbitmq`

Reply:
done